### DISCUSSION TITLE
Template - Github Actions Runner

### DISCUSSION THREAD

**Discussion by Vahor** [coollabsio/coolify#3187] [Service Template Requests]
I'm using actions to lint / test my code and build docker images.  But GitHub actions free-tier has some limit (cpu / quotas).
As we're using coolify we have servers, so why not use them instead, we're already using them as build servers.

Possible image / docker compose: https://github.com/myoung34/docker-github-actions-runner

Possibles improvements over a simple docker compose file:
- automatic tag based on the server
	- linux
	- x86 / arm64
	- gpu if enabled with checkbox
	- (might have to check if they're not set by default as I can see a `NO_DEFAULT_LABELS` env var)
- set `APP_PRIVATE_KEY`, `APP_ID`, `ORG_NAME`  ([env vars list](https://github.com/myoung34/docker-github-actions-runner?tab=readme-ov-file#environment-variables)) with Coolify Github app tokens.

**Comment by riemers** [comment]
i know late, but https://coolify.io/docs/services/github-runner.. its there but this was one of the first hits on google, hence i wanted to make the addition here.
